packages/core/src/combinator/withLocalTime.js

```diff
@@ -1,15 +1,24 @@
-/** @license MIT License (c) copyright 2010-2017 original author or authors */
-
 import RelativeSink from '../sink/RelativeSink'
 import { schedulerRelativeTo } from '@most/scheduler'
 
-// Run a stream with its own localized clock
+// Create a stream with its own local clock
```

Comment:
> ... with its own local clock

Do we also have a clock type? I know there is  `Time`.

Reply:
dah we do http://mostcore.readthedocs.io/en/latest/api.html#clock

Reply:
Mmmokay